File: practice/algorithms/sorting/intro_to_tutorial_challenges/IntroToTutorialChallenges.js
'use strict';

const processData = input => {
    let getPosition = (arr, val) => {
        for(let i = 0; i < arr.length; i++) {
            if(arr[i] == val) {
                return i;
            }
        }
    };
    let lines = input.split('\n');
    let searchedValue = parseInt(lines[0]);
    let array = lines[2].split(' ').map(i => parseInt(i));
    let position = getPosition(array, searchedValue);
    console.log(position);
};

process.stdin.resume();
process.stdin.setEncoding("ascii");

var _input = "";
process.stdin.on("data", input => _input += input);
process.stdin.on("end", () => processData(_input));

File: practice/algorithms/sorting/intro_to_tutorial_challenges/intro_to_tutorial_challenges.rb
def get_position(arr, val)
  (arr.size - 1).times do |i|
    return i if(val == arr[i])
  end
end

searched_value = gets.to_s.chomp.to_i
array_length = gets.to_s.chomp.to_i
arr = gets.to_s.split.map{ |i| i.to_i }

position = get_position(arr, searched_value)
puts position